candi.cfg

```diff
@@ -79,8 +79,8 @@ PACKAGES="${PACKAGES} dealii"
 
 #########################################################################
 
-# Install the following deal.II version:
-DEAL_II_VERSION=v9.2.0
+# Install the following deal.II version (choose master, v9.2.0 ...)
```

Comment:
Maybe use in the comment '... (choose master, v9.3.0, ...)' as v9.3.0 is already the newest stable one?